src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -23,6 +24,12 @@ public void actionPerformed(ActionEvent e) {
 			case "Figure Joe 2":
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
+			case "Factored triangle draw":
+				TestDrawer.factoredDraw(driverManager.getCurrentDriver(), TestDrawer.TypeFactoredOfTest.TRIANGLE);
+				break;
+			case "Factored rectangle draw":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.